doc/nrf/releases_and_maturity/migration/migration_guide_3.2.rst

```diff
@@ -62,6 +62,78 @@ Samples and applications
+
+     #. Since the MCUboot bootloader in the direct-xip mode uses a merged image slot for the ``nrf54h20dk/nrf54h20/cpuapp`` board target, define the custom memory layout in DTS (the ``partitions`` DTS node) and ensure that this DTS customization is propagated to every image that is built as part of the nRF Desktop application.
+        See the :file:`nrf/applications/nrf_desktop/configuration/nrf54lm20dk_nrf54lm20a_cpuapp/memory_map.dtsi` file for an example of the memory layout file and the :file:`nrf/applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/images/mcuboot/app.overlay` file for an example integration of the custom memory layout into the MCUboot bootloader image.
+        Apart from the MCUboot bootloader image, the custom memory layout must be defined in the following images:
+
+        * the ``ipc_radio`` image and its secondary variant.
+        * the ``nrf_desktop`` image (the default application image) and its secondary variant.
+        * the ``uicr`` image.
+
+        .. note::
+           The |NCS| v3.2.0 introduces a new image for the ``nrf54h20dk/nrf54h20/cpuapp`` board target called ``uicr``.
+           The custom memory layout must be defined in the ``uicr`` image as well to prevent runtime issues.
+           See the :file:`nrf/applications/nrf_desktop/configuration/nrf54h20dk_nrf54h20_cpuapp/images/uicr/app.overlay` file for an example integration of the custom memory layout into the ``uicr`` image.
+
+     #. Enable the following Kconfig options to improve USB report rate by suspending Bluetooth when USB is connected:
+
+        * :kconfig:option:`CONFIG_DESKTOP_BLE_ADV_CTRL_ENABLE` to ``y``.
+        * :kconfig:option:`CONFIG_DESKTOP_BLE_ADV_CTRL_SUSPEND_ON_USB` to ``y``.
+
+     #. Remove the usage of the following Kconfig options related to the S2RAM functionality:
+
+        * :kconfig:option:`CONFIG_PM_S2RAM`.
+        * :kconfig:option:`CONFIG_PM_S2RAM_CUSTOM_MARKING`.
+
+     #. Remove the ``pwmleds`` DTS node from the application configuration to prevent the :kconfig:option:`CONFIG_LED_PWM` and :kconfig:option:`CONFIG_CAF_LEDS_PWM` Kconfig options from being enabled for the ``nrf54h20dk/nrf54h20/cpuapp`` board target.
+
+        .. code-block::
+
+           / {
+             /delete-node/ pwmleds;
+
+             aliases {
+               /delete-property/ pwm-led0;
+             };
+           };
+
+     #. Disable Link Time Optimization (LTO) to prevent linker warnings about the type mismatch (``-Wlto-type-mismatch``) that are caused by the device_deps.c file (used by the Zephyr power domain driver):
+
+        * :kconfig:option:`CONFIG_LTO` to ``n``.
+        * :kconfig:option:`CONFIG_ISR_TABLES_LOCAL_DECLARATION` to ``n``.
+
+     #. Replace the ``interface-name`` property with the ``label`` property in all DTS nodes that set the ``compatible`` property to ``zephyr,hid-device``.
```

Comment:
I would suggest to mention that SUIT and IronSide application images are not interoperable (to let user know that he cannot update to IronSide in case of devices already deployed in field)